UOP1_Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.11f1
-m_EditorVersionWithRevision: 2019.4.11f1 (2d9804dddde7)
+m_EditorVersion: 2019.4.8f1
```

Comment:
We cannot change the project version.
You should not submitted this file. Please remove from your PR.

Reply:
My mistake. How can i remove it? I dont know